test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -187,12 +203,30 @@ describe("UserWallet - Lightning Pay", () => {
     expect(transaction1Below).toHaveProperty("description", memoSpamBelowThreshold)
 
     // check above-threshold transaction for recipient was NOT filtered
+    expect(transaction0Above.amount).toBeGreaterThanOrEqual(MEMO_SHARING_SATS_THRESHOLD)
     expect(transaction0Above).toHaveProperty("username", userWallet1.user.username)
     expect(transaction0Above).toHaveProperty("description", memoSpamAboveThreshold)
     expect(transaction1Above).toHaveProperty("username", userWallet0.user.username)
     expect(transaction1Above).toHaveProperty("description", memoSpamAboveThreshold)
 
+    // check funder below-threshold transaction for recipient was NOT filtered
+    const fundingUser = await User.findOne({ role: "funder" })
+    expect(fundingUser.username).toBe(userWallet4.user.username)
+    expect(transaction0FunderBelow.amount).toBeLessThan(MEMO_SHARING_SATS_THRESHOLD)
+    expect(transaction0FunderBelow.amount).toBeGreaterThanOrEqual(0)
+    expect(transaction0FunderBelow).toHaveProperty("username", userWallet4.user.username)
+    expect(transaction0FunderBelow).toHaveProperty(
+      "description",
+      memoFunderBelowThreshold,
+    )
+    expect(transaction4FunderBelow).toHaveProperty("username", userWallet0.user.username)
+    expect(transaction4FunderBelow).toHaveProperty(
+      "description",
+      memoFunderBelowThreshold,
+    )
+
     // check contacts being added
+    // TODO: decide if we want to exclude spam senders from contacts
```

Comment:
this is a TODO but not in test, seems to be more a question for #dev channel in slack. Lets try to keep the code as clean as possible.